cross/numpy/Makefile

```diff
@@ -0,0 +1,16 @@
+PKG_NAME = numpy
+PKG_VERS = 1.17.5
+PKG_EXT = zip
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/n/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = https://numpy.org/
+COMMENT  = The fundamental package for scientific computing with Python
+LICENSE  = https://numpy.org/doc/stable/license.html
+
+ADDITIONAL_CFLAGS = -std=c99
```

Comment:
This is why a mere requirements file couldn't be used: compilation needed to be customized to pass.